lib/shelley/bench/Latency.hs

```diff
@@ -151,11 +152,8 @@ walletApiBench capture ctx = do
     fmtTitle "Latencies for 10 fixture wallets scenario"
     runScenario (nFixtureWallet 10)
 
-    {-- PENDING: We currently have a limited amount of available fixture
-       wallets, so we can't just run a benchmark with 100 wallets in parallel.
-    fmtTitle "Latencies for 100 fixture wallets scenario"
+    fmtTitle "Latencies for 100 fixture wallets"
     runScenario (nFixtureWallet 100)
```

Comment:
Seems a bit low:
```
 Latencies for 100 fixture wallets
        listWallets         - 59.9 ms
        getWallet           - 0.5 ms
        getUTxOsStatistics  - 0.4 ms
        listAddresses       - 1.2 ms
        listTransactions    - 2.6 ms
        postTransactionFee  - 139.2 ms
        listStakePools      - 1.1 ms
        getNetworkInfo      - 0.0 ms
```

I wondered if ensuring the wallets were actively restoring (by replacing `fixtureWallet` with `emptyWallet`, which doesn't wait for funds to be available, so should be quicker), and it did get _a bit_ slower:
```
    Latencies for 100 fixture wallets
        listWallets         - 71.0 ms
        getWallet           - 0.5 ms
        getUTxOsStatistics  - 0.4 ms
        listAddresses       - 0.7 ms
        listTransactions    - 0.1 ms
        postTransactionFee  - 263.6 ms
        listStakePools      - 1.3 ms
        getNetworkInfo      - 0.1 ms
```
I want to test on mainnet though.

Reply:
Btw, it's in the context of this (estimate fees taking minutes with 100 wallets)
<img width="991" alt="Skärmavbild 2020-10-08 kl  12 12 30" src="https://user-images.githubusercontent.com/304423/95445793-b3eb0880-095f-11eb-9f06-11058a8f1eff.png">
that sub-second results seem suspicious.

Reply:
It oddly long, especially for wallets that are empty. If you want to spend some time investigating this, please do. Don't spend _too much_ time on this however (max 1 day). It you can't find anything conclusive, then we'll allocate some extra time where finding the root cause and fixing it will be the actual task / goal.